UI/css/style.css

```diff
@@ -99,25 +99,29 @@ button{
      box-shadow: 0 0 35px rgba(0, 0, 0, .5);
 }
 
-button:hover {
-     color: green;
+.home-button:hover {
      background: whitesmoke;
+     color: green;
 }
 
-button a {
-      text-decoration: none;
-      color:whitesmoke;
-      background: green;
+.home-button a {
+     text-decoration: none;
+     color:whitesmoke;
+     background: none;
  }
-button a:hover {
+.home-button a:hover {
      color: green;
-     background-color: whitesmoke;
 }
 
 footer {
      min-height: 10vh;
+     width: 100%;
      clear: both;
-     padding-top: 3%;
+     padding-top: 3%;     
```

Comment:
@ebenezeradukunda use padding in ``px` instead of `%`

Reply:
Thanks for this, I have ready done it, it looks nice

![Footer](https://user-images.githubusercontent.com/54438606/69666020-36ded980-1094-11ea-9e5a-d8b1727c86a5.PNG)